common/beerocks/bcl/include/bcl/beerocks_defines.h

```diff
@@ -43,7 +43,7 @@ enum eStructsConsts {
     VERSION_LENGTH            = 16,
     NODE_NAME_LENGTH          = 32,
     IFACE_NAME_LENGTH         = 32 + 4, //need extra 1 byte for null termination + alignment
-    SUPPORTED_CHANNELS_LENGTH = 64,     //support upto # channels, every channel item is 32-bit
+    SUPPORTED_CHANNELS_LENGTH = 80,     //support upto # channels, every channel item is 32-bit
```

Comment:
didn't we agree on 128 ?

Reply:
When I expanded the size to 128, it failed the `test_flows` the same way currently the 80 fails with the `test-on-netgear-rax40`

I believe the tests fail due to an issue with the AP_JOINED and M1 messages, as I see (in both cases) that the agent is stuck in a never-ending loop of `STATE_WAIT_FOR_JOINED_RESPONSE`

Reply:
That's probably because sNodeHostap becomes too large. With 128 channels, the preferred channel list in it becomes 1K so it's very easy to reach the maximum TLV size.

The best solution would be to remove preferred channels from sNodeHostap completely and instead use the channel preference flow. But that's more work, I guess.

Reply:
yes we should find a w/a for now.
@itayx  please  check what can we remove from supported_channels for now that is not used.
also go over sNodeHostap parameters and see if there is a something we can remove.
If nothing can be removed i propose removing them from sNodeHostap and create new VS CMDU only from supported channels.